packages/m/mozc/package.yml

```diff
@@ -29,38 +35,41 @@ rundeps    :
         - mozc
     - ^ibus-mozc :
         - mozc
-patterns   :
-    - ^fcitx-mozc :
-        - /usr/lib/fcitx
-        - /usr/share/fcitx
-    - ^ibus-mozc :
-        - /usr/lib/ibus-mozc
-        - /usr/share/ibus
 setup      : |
-    %patch -p1 -i $pkgfiles/gcc.patch
-    rm -rf src/unix/fcitx5
+    %apply_patches
+
     pushd ..
-    tar xf $sources/fcitx-mozc-icon.tar.gz
+    unzip $sources/ken_all.zip
+    unzip $sources/jigyosyo.zip
     popd
 
-    pushd src/third_party/gyp
-    git cherry-pick -n bc83cdacf5428ab6ddcc92fd0b0fc494cc9a4d4f
+    # Seed zip code dictionary generator
+    pushd src
```

Comment:
Maybe just one `pushd src/data/dictionary_oss` and `popd`?

Reply:
I didn't know you could do that. That's way better!